test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -1527,6 +1614,44 @@ public void testGetDocIds_urlAndMetadataLister() throws Exception {
         pusher.getRecords());
   }
 
+  @Test
```

Comment:
Another test that I don't see from before that we could have had, is to make sure that getDocIds and getModifiedDocIds really do ignore the metadata config when docIdIsUrl. We have lots of init tests, and one each for getDocIds and getModifiedDocIds, to check the URL and date handling. I think we should either add some metadata to those, or have a new pair of tests with just a single row, and both metadataColumns and extraMetadataSql configured, and verify that no metadata is returned (and maybe extraMetadataSql could be syntactically invalid; though I think we "ignore" the property, but still validate the SQL?).